src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -223,15 +223,16 @@ public long LastRaftCommitIndex
                                         if (notification == null)
                                             break;
 
-                                        if (notification.TryGet("Id",out string id) && id != nameof(ClusterTopologyChanged))
+                                        if (notification.TryGet(nameof(ClusterTopologyChanged.Type), out NotificationType notificationType) == false ||
```

Comment:
Here too, we need the backoff strategy when calling `UpdateTopologyChangeNotification`